changelog/rdmd_shared.dd

```diff
@@ -0,0 +1,3 @@
+rdmd supports `-shared`
+
+rdmd now understands DMD's `-shared` switch, and sets the default output file name appropriately (`.dll` or `so` depending on the platform), in the same way as `-lib`.
```

Comment:
I missed one on the changelog too, not that it really matters. 

Feel free to merge when green.